### DISCUSSION TITLE
cn/2020/01/19/hugo-google-noto-serif-sc/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#162] [Announcements]
# cn/2020/01/19/hugo-google-noto-serif-sc/

https://hongtaoh.com/cn/2020/01/19/hugo-google-noto-serif-sc/

**Comment by chuxinyuan** [comment]
关于字体，最近算是有点心得了，按照[益辉的方法](https://github.com/yihui/yihui.org/discussions/1547)自己编制博客需要的字体就好，非常轻量。
